pickhardtpayments/OracleChannel.py

```diff
@@ -37,3 +37,7 @@ def can_forward(self, amt: int):
             return True
         else:
             return False
+
+    # setter for actual liquidity
+    def set_actual_liquidity(self, amt: int):
```

Comment:
the python why is not to do setter and getter functions but use the mechanism that I did in the `UncertaintyChannel` class using decorators. look for lines that have `@property` to define getter and `@....setter` to define setter

Reply:
fix in afa6a71